dev/src/components/ReviewPage.tsx

```diff
@@ -41,26 +43,22 @@ const ReviewPage = (): JSX.Element => {
         }
     }
 
-    const header = ["Name"].concat(state.assignments[courseID].map(assignment => assignment.getName()))
-
-
-    const data = state.courseSubmissions[courseID]?.map((link) => {
-        const row: Row = []
-        if (link.submissions && link.user) {
-            row.push({ value: link.user.getName(), link: `https://github.com/${link.user.getLogin()}` })
-            link.submissions.forEach(submission => {
-                row.push(generateReviewCell(submission))
-            })
-        }
-        return row
-    })
+    const links = groupView ? state.courseGroupSubmissions[courseID] : state.courseSubmissions[courseID]
+    const rows = generateSubmissionRows(links, generateReviewCell)
```

Comment:
In `Results.tsx` it has an extra `true` argument; should that be done here also?: (Didn't check the logic, just noticed the difference.)
```ts
const results = generateSubmissionRows(links, getSubmissionCell, true)
```